platform/tests/stacks/up/up_test.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+test_setup() {
```

Comment:
@appcelerator/amp Just be aware that the test runner doesn't have formal support for test_setup/test_teardown; this works right now because the test runner doesn't currently run tests in parallel. Most developers will assume test_setup and test_teardown is something that runs before and after *each* test; for something to run just once for the file, we will need to add `setup()` and `teardown()` support (but those won't match the current pattern for `test_*()`).